ipfsconn/ipfshttp/ipfshttp.go

```diff
@@ -612,6 +617,48 @@ func (ipfs *Connector) RepoStat(ctx context.Context) (*api.IPFSRepoStat, error)
 	return &stats, nil
 }
 
+// RepoGC performs a garbage collection sweep on the repo.
+func (ipfs *Connector) RepoGC(ctx context.Context) (*api.RepoGC, error) {
+	ctx, span := trace.StartSpan(ctx, "ipfsconn/ipfshttp/RepoGC")
+	defer span.End()
+
+	id, err := ipfs.ID(ctx)
+	if err != nil {
+		logger.Error(err)
+		return nil, err
+	}
+	repoGC := api.RepoGC{
+		Peer: id.ID,
```

Comment:
I thought we'd be setting this to the Cluster peer ID (in `cluster.go`). If you want to carry the ipfs id, you'll need an additional field in the `RepoGC`.

Reply:
Uses cluster peer ID now